go.mod

```diff
@@ -2,24 +2,16 @@ module github.com/juju/mgomonitor
 
 require (
 	github.com/beorn7/perks v0.0.0-20180321164747-3a771d992973 // indirect
-	github.com/cloud-green/monitoring v0.0.0-20180917115443-469b6e954230
 	github.com/frankban/quicktest v1.1.0
 	github.com/golang/protobuf v1.2.0 // indirect
 	github.com/google/go-cmp v0.2.0
-	github.com/juju/clock v0.0.0-20180808021310-bab88fc67299 // indirect
-	github.com/juju/errors v0.0.0-20181012004132-a4583d0a56ea // indirect
-	github.com/juju/loggo v0.0.0-20180524022052-584905176618 // indirect
-	github.com/juju/retry v0.0.0-20180821225755-9058e192b216 // indirect
-	github.com/juju/testing v0.0.0-20180920084828-472a3e8b2073
-	github.com/juju/utils v0.0.0-20180820210520-bf9cc5bdd62d // indirect
-	github.com/juju/version v0.0.0-20180108022336-b64dbd566305 // indirect
 	github.com/matttproud/golang_protobuf_extensions v1.0.1 // indirect
-	github.com/prometheus/client_golang v0.9.0
+	github.com/prometheus/client_golang v0.0.0-20161124155732-575f371f7862
```

Comment:
It'd be nice to stick with what I assume was a newer version.

Reply:
I'm using the same version as used by the charmstore. The newer version pulled in some deps I wasn't sure about and I know that prometheus doesn't always keep backward compatibility, so best to be safe I think.